rules/windows/execution_from_unusual_directory.toml

```diff
@@ -0,0 +1,41 @@
+[metadata]
+creation_date = "2020/10/30"
+maturity = "production"
+updated_date = "2020/10/30"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies process execution from suspicious default Windows directories. This is sometimes done by
+adversaries to hide malware in trusted paths.
+"""
+from = "now-9m"
+index = ["winlogbeat-*", "logs-endpoint.events.*"]
+language = "eql"
+license = "Elastic License"
+name = "Process Execution from an Unusual Directory"
+risk_score = 47
+rule_id = "ebfe1448-7fac-4d59-acea-181bd89b1f7f"
+severity = "medium"
+tags = ["Elastic", "Host", "Windows", "Threat Detection", "Execution"]
+type = "eql"
+
+query = '''
```

Comment:
yet to test this query in a dev stack in case it is too resource intensive. will need to spin up a local master build and test before approving.

Reply:
for me, works with no issues let me know if u observe any issues .. 

![image](https://user-images.githubusercontent.com/64742097/101209913-e6219980-3674-11eb-867a-8380df33e6b0.png)

Reply:
tested with no issues as well